persistence/trees/trees_test.go

```diff
@@ -0,0 +1,19 @@
+package trees
+
+import "testing"
+
+// TECHDEBT #836: Tests added in https://github.com/pokt-network/pocket/pull/836
```

Comment:
```suggestion
// TECHDEBT(#836): Tests added in https://github.com/pokt-network/pocket/pull/836
```

I'm just thinking about automation and regex parsing